.github/workflows/gradle-build-feature.yml

```diff
@@ -0,0 +1,19 @@
+name: Gradle Build Feature Branch
```

Comment:
Should this (line 1) say something about bugs? Like Gradle Build Feature/Bug Branch?